src/test/groovy/com/calgaryscientific/gradle/VeracodeWorkflowDOTest.groovy

```diff
@@ -0,0 +1,216 @@
+/*******************************************************************************
+ * MIT License
+ *
+ * Copyright (c) 2017 Calgary Scientific Incorporated
+ *
+ * Copyright (c) 2013-2014 kctang
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ ******************************************************************************/
+
+package com.calgaryscientific.gradle
+
+import org.gradle.api.Project
+import org.gradle.testfixtures.ProjectBuilder
+
+class VeracodeWorkflowDOTest extends TestCommonSetup {
+    // Status after just creating a new build
+    File buildInfoFileIncomplete = getResource('buildinfo-1.4-incomplete.xml')
+    // Status after Scan submitted
+    File buildInfoFile = getResource('buildinfo-1.4.xml')
+    // Status after pre scan is completed
+    File buildInfoFilePreScanSuccess = getResource('buildinfo-1.4-preScanSuccess.xml')
+    // Status after scan completed
+    File buildInfoFileResultsReady = getResource('buildinfo-1.4-complete.xml')
+
+    File filelistFile = getResource('filelist-1.1.xml')
+    File preScanResultsFile = getResource('prescanresults-1.4.xml')
+
+    def 'Test appWorkflow when previous build has results ready'() {
```

Comment:
typically, the test names should some how reflect the expect behavior or examples you're trying to accomplish .... so if it is named something like given previous build results ready then work flow should start scan ....., this case I can see what you are trying to accomplish.....